ietf/bin/iana-protocols-updates

```diff
@@ -31,8 +31,11 @@ syslog.syslog("Updating history log with new RFC entries from IANA protocols pag
 # FIXME: this needs to be the date where this tool is first deployed
 rfc_must_published_later_than = datetime.datetime(2012, 11, 26, 0, 0, 0)
 
-text = requests.get(settings.IANA_SYNC_PROTOCOLS_URL).text
-rfc_numbers = parse_protocol_page(text)
+response = requests.get(
+    settings.IANA_SYNC_PROTOCOLS_URL,
+    timeout=30,
+)
```

Comment:
The addition of the timeout opens the possibility of raising a requests.exceptions.Timeout exception. It would be nice to handle that, though it would be understandable if you decide that not erroring out of the command line utility is a correct behavior.